modules/installation-vsphere-requirements-user-provisioned.adoc

```diff
@@ -0,0 +1,258 @@
+`VirtualMachine.Interact.GuestControl`
+`VirtualMachine.Interact.PowerOff`
+`VirtualMachine.Interact.PowerOn`
+`VirtualMachine.Interact.Reset`
+`VirtualMachine.Inventory.Create`
+`VirtualMachine.Inventory.CreateFromExisting`
+`VirtualMachine.Inventory.Delete`
+`VirtualMachine.Provisioning.Clone`
+
+|vSphere vCenter Datacenter
+|If the installation program creates the virtual machine folder
+|
+[%hardbreaks]
+`Resource.AssignVMToPool`
+`VApp.Import`
+`VirtualMachine.Config.AddExistingDisk`
+`VirtualMachine.Config.AddNewDisk`
+`VirtualMachine.Config.AddRemoveDevice`
+`VirtualMachine.Config.AdvancedConfig`
+`VirtualMachine.Config.Annotation`
+`VirtualMachine.Config.CPUCount`
+`VirtualMachine.Config.DiskExtend`
+`VirtualMachine.Config.DiskLease`
+`VirtualMachine.Config.EditDevice`
+`VirtualMachine.Config.Memory`
+`VirtualMachine.Config.RemoveDisk`
+`VirtualMachine.Config.Rename`
+`VirtualMachine.Config.ResetGuestInfo`
+`VirtualMachine.Config.Resource`
+`VirtualMachine.Config.Settings`
+`VirtualMachine.Config.UpgradeVirtualHardware`
+`VirtualMachine.Interact.GuestControl`
+`VirtualMachine.Interact.PowerOff`
+`VirtualMachine.Interact.PowerOn`
+`VirtualMachine.Interact.Reset`
+`VirtualMachine.Inventory.Create`
+`VirtualMachine.Inventory.CreateFromExisting`
+`VirtualMachine.Inventory.Delete`
+`VirtualMachine.Provisioning.Clone`
+`VirtualMachine.Provisioning.DeployTemplate`
+`VirtualMachine.Provisioning.MarkAsTemplate`
+`Folder.Create`
+`Folder.Delete`
+|===
+====
+
+Additionally, the user requires some `ReadOnly` permissions, and some of the roles require permission to propagate the permissions to child objects. These settings vary depending on whether or not you install the cluster into an existing folder.
+
+.Required permissions and propagation settings
+[%collapsible]
+====
+[cols="3a,3a,3a,3a",options="header"]
+|===
+|vSphere object
+|Folder type
+|Propagate to children
+|Permissions required
+
+|vSphere vCenter
+|Always
+|False
+|Listed required privileges
+
+.2+|vSphere vCenter Datacenter
+|Existing folder
+|False
+|`ReadOnly` permission
+
+|Installation program creates the folder
+|True
+|Listed required privileges
+
+|vSphere vCenter Cluster
+|Always
+|True
+|Listed required privileges
+
+|vSphere vCenter Datastore
+|Always
+|False
+|Listed required privileges
+
+|vSphere Switch
+|Always
+|False
+|`ReadOnly` permission
+
+|vSphere Port Group
+|Always
+|False
+|Listed required privileges
+
+|vSphere vCenter Virtual Machine Folder
+|Existing folder
+|True
+|Listed required privileges
+|===
+====
+
+[discrete]
+[id="installation-vsphere-requirements-user-provisioned-machine-sets_{context}"]
+== Minimum required vCenter privileges for machine set management
+
+The following table lists the minimum roles and privileges that are required to create, scale and delete machine sets and to delete machines in your cluster after installation.
+
+.Minimum roles and privileges required to create, scale and delete machine sets
```

Comment:
```suggestion
.Minimum roles and privileges required to create, scale, and delete machine sets
```

Reply:
@mjpytlak thank you for your review and for highlighting this. Resolved.